src/crimson/tools/store_bench/store-bench.cc

```diff
@@ -676,10 +673,12 @@ seastar::future<results_t> RandomWriteWorkload::run(
    coll_refs.emplace_back(std::make_pair(cid, std::move(ref)));
   }
   auto get_coll_id = [&](uint64_t obj_id) {
-    return coll_refs[obj_id % get_obj_per_coll()].first;
+    assert(obj_id % colls_per_shard < coll_refs.size());
```

Comment:
Can you please group this in parentheses?

Reply:
Done